### PR TITLE
Add API enabled retail post

### PR DESCRIPTION
**Pull request by chrisblackburn**

### CODE REVIEW

File: posts/api_enabled_retail.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Modern APIs are Pushing Innovation in Retail
+
+An increasing number of retailers are API enabling core parts of their business, delivering a more cohesive customer experience across a variety of touchpoints, making it easier to streamline internal operations, and opening opportunities to interface with external channels.
```
Comment (zuren) [inline posts/api_enabled_retail.md]: "API enabling" is a weird phrase IMO. "exposing core parts of their business through APIs"?

File: posts/api_enabled_retail.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## What is an API?
+
+First off, it's worth recapping on what we mean by API, or Application Programming Interface, to use its full name.
```
Comment (sowasred2012) [inline posts/api_enabled_retail.md]: Don't need the word "on".

File: posts/api_enabled_retail.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+An API allows systems to easily talk to one another. APIs can allow an e-commerce store to send an order to the warehouse for fulfilment, or an EPOS system understand what stock is available in the store.
```
Comment (zuren) [inline posts/api_enabled_retail.md]: EPOS system _to_ understand

Comment (sowasred2012) [inline posts/api_enabled_retail.md]: an EPOS system *to* understand

File: posts/api_enabled_retail.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+More modern applications are often API ready out of the box, but legacy applications can sometimes cause sticking points.
+
+## Legacy Applications and Proprietary Protocols
+
+Historically, applications connected via proprietary protocols. Different vendors had their own standards, and connecting different systems was often a laborious exercise that involved licensing more proprietary software.
+
+Worse is that some software systems are "walled gardens", where they by design, or through lack of investment, won't interoparate with other systems. Isolating their data and functionality from the rest of the organisation.
```
Comment (zuren) [inline posts/api_enabled_retail.md]: "_This isolates_ their data and functionality from the rest of the organisation" ?

Comment (sowasred2012) [inline posts/api_enabled_retail.md]: I would shift some commas here: ...are "walled gardens" where they, by design or through lack of investment, won't...

Also: interoperate typo

File: posts/api_enabled_retail.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+In today's world, customers are demanding a cohesive experience that this isolationist approach can hinder.
+
+## Customers Increasingly Expect a More Connected Experience
+
+Customers are interacting with retailers through an increasing array of touchpoints, from web, to mobile, and in-store, and perhaps third party marketplaces.
```
Comment (sowasred2012) [inline posts/api_enabled_retail.md]: I think "and perphaps third party" might read better as "perhaps even third party"

File: posts/api_enabled_retail.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+An organisation that has adopted an API ready approach will find integrating new channels a much easier experience. If the e-commerce platform already exposes an API for accepting payments, half the job would already be done in allowing a new mobile application the same functionality. Likewise if the warehouse system already provides an API for stock levels, showing live stock data on the e-commerce storefront is largely solved.
+
+Centralising customer data in to a CRM system that can be accessed via APIs allows a customer to be more easily recognised across the retail touchpoints, and can enable the shopping experience to be tailored to them. A salesperson who can see a customers recent browsing habits and online purchases is much better able to make recommendations. Likewise, if the data feeds both ways, customers can be retargeted with relevant advertising online based on their offline behaviour.
```
Comment (sowasred2012) [inline posts/api_enabled_retail.md]: "a customer's recent browsing habits"

File: posts/api_enabled_retail.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Many retailers already have significant maturity in delivering APIs
```
Comment (sowasred2012) [inline posts/api_enabled_retail.md]: Is this line orphaned from notes?

File: posts/api_enabled_retail.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+On a smaller scale, startups are entering many industries and adopting or building modern, API enabled software to do so. Payment processor Stripe has significantly lowered the barrier to entry in processing online payments, in part by how easy they've made it to integrate with your application. Algolia offers a search platform bettering many existing enterprise offerings, enabling developers to be up and running within an hour.
+
+Offering APIs can open opportunities outside of the organisation too.
+
+## Retail APIs and Third Party Services
+
+Integrating with third party services easily can be a compelling benefit. Enabling commerce anywhere (think "shop the look" on Instagram), or exposing product catalogues to price comparison or aggregation services without the need for a standalone channel manager to power an integration.
+
+Opportunities can be opened to work with partner organisations with complimentary offerings, or to encourage third party developers to interface with your systems. Reportedly 60% of eBay's revenue is driven through API calls, and for Expedia the number is reportedly as high as 90%.
```
Comment (sowasred2012) [inline posts/api_enabled_retail.md]: Repeated reportedly in that last line - could also be a good place to slot a link in, with a reference to where those stats come from?

**Pull request merged**